lms/static/js/commerce/views/receipt_view.js

```diff
@@ -47,6 +47,8 @@ var edx = edx || {};
 
             this.renderCourseNamePlaceholder(this.courseKey);
 
+            this.renderUserFullNamePlaceholder(this.username);
```

Comment:
Why do you need these function which update the DOM after you have compiled the template? It would make more sense to compile the template with all of the data present so you don't need these further DOM updates.